pkg/addons/addons.go

```diff
@@ -176,6 +176,16 @@ https://github.com/kubernetes/minikube/issues/7332`, out.V{"driver_name": cc.Dri
 		return nil
 	}
 
+	if name == "registry" {
+		port, err := machine.RegistryPort(cc)
+		if err != nil {
+			errors.Wrap(err, "Can't get Registry port")
+		}
+		if runtime.GOOS != "linux" {
+			out.T(out.Notice, `This is the port you should use for the registry addon "{{.port}}"`, out.V{"port": port})
```

Comment:
change this wording to:
`   registry addon on with {{.driver}} uses {{.port}} please use that
for more information see
"https://minikube.sigs.k8s.io/docs//link to documentation"

(and add a documentaiton to the website)